test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature

```diff
@@ -14,25 +14,27 @@ Feature: Status Desktop Sign Up
   As a user I want to Sign-up into the Status Desktop application.
   The following scenarios cover Sign up process.
 
+  @journey-test
   Scenario: User signs up and signs in with password
     Given A first time user lands on the status desktop and generates new key
     When user signs up with username tester123 and password TesTEr16843/!@00
     Then the user lands on the signed in app
 
-
+  @journey-test
   Scenario Outline: User cannot sign up with wrong username format
     Given A first time user lands on the status desktop and generates new key
     When user inputs the following <username> with ui-component mainWindow_edit_TextEdit
```

Comment:
This statement should be avoided and encapsulate the needed steps in a specific screen definition. Same for the next statements.

Reply:
Yeah, but I mean in this particular case I see that it could be better to encapsulate the functionality into a specific screen object. WDYT?